pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -346,7 +351,11 @@ func ConstructObject(object v1.CustomObject, appObject v1.KieAppObject) v1.Custo
 }
 
 // getKieSetName aids in server indexing, depending on number of deployments and sets
-func getKieSetName(applicationName string, setName string, arrayIdx, deploymentsIdx int) string {
+func getKieSetName(applicationName string, setName string, arrayIdx int) string {
+	return getKieDeploymentName(applicationName, setName, arrayIdx, 0)
+}
+
+func getKieDeploymentName(applicationName string, setName string, arrayIdx, deploymentsIdx int) string {
```

Comment:
_getKieDeploymentName_ is a little confusing as it infer's the deployment/DC name. If this is actually the kieserver name, should we change the function name to `getKieServerName`? I assume this name will be used for all kieserver objects right?

Reply:
Well, as we have the concept of `deployments` I thought it will more appropriate. To distinguish it between kieServerSet (`serverSet.Name`) and kieServerDeployment (`myapp-kieserver2-3`).